test/cypress-e2e/cypress/e2e/5-group-renewal-application/group-renewal-application-full-e2e.cy.js

```diff
@@ -0,0 +1,19 @@
+import { loginPage } from '../../support/pages/1-portal-login-pages/01-portal-login.js'
+import { submitApp } from '../../support/pages/2-portal-application-pages/04-portal-application-summary-declaration.js'
+
+
+describe('Group Renewal Application Test', () => {
+    it('Should run through Group Renewal Application, submit and logout', () => {
+        loginPage.visitLoginPage()
+        loginPage.clickLoginButton()
+        loginPage.loginThroughExternalProvider(
+            Cypress.env("PORTAL_USERNAME"),
+            Cypress.env("PORTAL_PASSWORD"))
+        cy.startNewRenewalApp()
+        cy.licenceUpload()
+        cy.runCcfriApp('groupRenewal', '2026-27')
```

Comment:
I'm seeing this value (and other such as familyRenewal) enough that you might want to consider a shared constant for it within your test folder. You could add /support/constants.js. Have a look at frontend/src/utils/constants.js for an option to structure then as Objects.